Marlin/src/lcd/language/language_de.h

```diff
@@ -151,65 +154,66 @@
+#define MSG_USERWAIT                        _UxGT("Klick zum fortsetzen")
 #define MSG_PRINT_PAUSED                    _UxGT("Druck pausiert")
+#define MSG_PRINTING                        _UxGT("Druckt...")
 #define MSG_PRINT_ABORTED                   _UxGT("Druck abgebrochen")
-#define MSG_NO_MOVE                         _UxGT("Motoren eingeschaltet")
+#define MSG_NO_MOVE                         _UxGT("Motoren einschalten")
 #define MSG_KILLED                          _UxGT("ABGEBROCHEN")
 #define MSG_STOPPED                         _UxGT("ANGEHALTEN")
-#define MSG_CONTROL_RETRACT                 _UxGT("Retract mm")
-#define MSG_CONTROL_RETRACT_SWAP            _UxGT("Wechs. Retract mm")
-#define MSG_CONTROL_RETRACTF                _UxGT("Retract  V")
-#define MSG_CONTROL_RETRACT_ZLIFT           _UxGT("Z-Hop mm")
+#define MSG_CONTROL_RETRACT                 _UxGT("Einzug mm")
+#define MSG_CONTROL_RETRACT_SWAP            _UxGT("Wechs. Einzug mm")
+#define MSG_CONTROL_RETRACTF                _UxGT("Einzug   V")
+#define MSG_CONTROL_RETRACT_ZLIFT           _UxGT("Z-Sprung mm")
 #define MSG_CONTROL_RETRACT_RECOVER         _UxGT("UnRet mm")
 #define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("Wechs. UnRet mm")
 #define MSG_CONTROL_RETRACT_RECOVERF        _UxGT("UnRet  V")
 #define MSG_CONTROL_RETRACT_RECOVER_SWAPF   _UxGT("S UnRet V")
-#define MSG_AUTORETRACT                     _UxGT("Autom. Retract")
+#define MSG_AUTORETRACT                     _UxGT("Autom. Einzug")
 #define MSG_FILAMENTCHANGE                  _UxGT("Filament wechseln")
 #define MSG_FILAMENTLOAD                    _UxGT("Filament laden")
 #define MSG_FILAMENTUNLOAD                  _UxGT("Filament entladen")
 #define MSG_FILAMENTUNLOAD_ALL              _UxGT("Alles entladen")
 #define MSG_INIT_SDCARD                     _UxGT("SD-Karte erkennen")  // Manually initialize the SD-card via user interface
 #define MSG_CNG_SDCARD                      _UxGT("SD-Karte getauscht") // SD-card changed by user. For machines with no autocarddetect. Both send "M21"
-#define MSG_ZPROBE_OUT                      _UxGT("Sensor ausserhalb")
-#define MSG_SKEW_FACTOR                     _UxGT("Skew Faktor")
+#define MSG_ZPROBE_OUT                      _UxGT("Z-Sonde ausserhalb")
+#define MSG_SKEW_FACTOR                     _UxGT("Korrektur Faktor")
 #define MSG_BLTOUCH                         _UxGT("BLTouch")
 #define MSG_BLTOUCH_SELFTEST                _UxGT("BLTouch Test")
-#define MSG_BLTOUCH_RESET                   _UxGT("BLTouch Reset")
+#define MSG_BLTOUCH_RESET                   _UxGT("BLTouch zurücks.")
 #define MSG_BLTOUCH_DEPLOY                  _UxGT("BLTouch ausfahren")
 #define MSG_BLTOUCH_STOW                    _UxGT("BLTouch einfahren")
-#define MSG_MANUAL_DEPLOY                   _UxGT("Z-Sensor anbringen")
-#define MSG_MANUAL_STOW                     _UxGT("Z-Sensor entfernen")
+#define MSG_MANUAL_DEPLOY                   _UxGT("Z-Sonde anbringen")
```

Comment:
"Sonde" is an uncommon word in that context, i would prefere "Sensor"

Reply:
Ok nach der Diskussion würde ich dann jetzt überall Z-Sonde schreiben, damit es einheitlich bleibt.